tests/phpunit/PlaceholdersUtilsTest.php

```diff
@@ -118,4 +118,326 @@ public function placeholderValueProvider(): array
+                    [
+                        'placeholder' => '1:id',
+                        'field' => 'id',
+                        'value' => 123,
+                    ],
+                ],
+            ],
+        ];
+
+        yield 'one-array' => [
+            [
+                [
+                    'placeholder' => '1:id',
+                    'field' => 'id',
+                    'value' => [123, 456, 789],
+                ],
+            ],
+            [
+                // 3 child job with 1 parameter
+                [
+                    [
+                        'placeholder' => '1:id',
+                        'field' => 'id',
+                        'value' => 123,
+                    ],
+                ],
+                [
+                    [
+                        'placeholder' => '1:id',
+                        'field' => 'id',
+                        'value' => 456,
+                    ],
+                ],
+                [
+                    [
+                        'placeholder' => '1:id',
+                        'field' => 'id',
+                        'value' => 789,
+                    ],
+                ],
+            ],
+        ];
+
+        yield 'scalar+array' => [
+            [
+                [
+                    'placeholder' => '1:category',
+                    'field' => 'category',
+                    'value' => 'other',
+                ],
+                [
+                    'placeholder' => '1:id',
+                    'field' => 'id',
+                    'value' => [123, 456, 789],
+                ],
+            ],
+            [
+                // 3 child job with 2 parameters
+                [
+                    [
+                        'placeholder' => '1:category',
+                        'field' => 'category',
+                        'value' => 'other',
+                    ],
+                    [
+                        'placeholder' => '1:id',
+                        'field' => 'id',
+                        'value' => 123,
+                    ],
+                ],
+                [
+                    [
+                        'placeholder' => '1:category',
+                        'field' => 'category',
+                        'value' => 'other',
+                    ],
+                    [
+                        'placeholder' => '1:id',
+                        'field' => 'id',
+                        'value' => 456,
+                    ],
+                ],
+                [
+                    [
+                        'placeholder' => '1:category',
+                        'field' => 'category',
+                        'value' => 'other',
+                    ],
+                    [
+                        'placeholder' => '1:id',
+                        'field' => 'id',
+                        'value' => 789,
+                    ],
+                ],
+            ],
+        ];
+
+        yield 'multiple' => [
```

Comment:
Extremny pripad, viacero skalarnych a array hodnot -> musia sa spravit kombinacie, ... v praxi to asi az taketo rozsiahle nikdy nebude.